ui-react/src/index.css

```
@@ -0,0 +1,55 @@
+@tailwind base;
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)